site/en/articles/renderingng-fragmentation/index.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: 'layouts/blog-post.njk'
```

Comment:
Please make sure there is no hero image, like the other RenderingNG articles, and that the social media share metadata indicates Una's picture as the one to go with the link.